hedyorg/hedy
# [FIX, UI] Editor size depending on parent

**Description**

Sets the size of the editor depending on its parent, `#editor`, when creating it. Also changes the implementation of the `resize` function to change the `height` of the `style` property of the editor's dom element, this simplifies the constructor a little bit. 

**Fixes #4841 and fixes #4866 (because that the buttons weren't shown was actually a fault of this) **

**How to test**

* Go to the slides and see that the editor is shown correctly
* Go to: http://0.0.0.0:8080/embedded/1/?lang=nl&run=true&readOnly=false&program=cHJpbnQgaGFsbG8gd2VyZWxkIQ==

And check that the editor takes the entire space.
